components/google_sign_in/browser/test/BUILD.gn

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//testing/test.gni")
+
+if (!is_android) {
+  source_set("browser_tests") {
```

Comment:
browser tests should not be in `components`, please move it to `//brave/browser/permissions` or something similar. This is a common mistake in our codebase, we shouldn't create new instances.

Basically, any `//chrome/browser` is forbidden in `//components`, which automatically forbids chrome browser tests here.
`content::RenderViewTest` are fine, but they should be located in `component/<name>/content` dir only.